recipe/meta.yaml

```diff
@@ -77,6 +77,9 @@ outputs:
         - {{ cdt('libxdamage-devel') }}      # [linux]
         - {{ cdt('libxfixes-devel') }}       # [linux]
         - {{ cdt('libxxf86vm-devel') }}      # [linux]
+        - {{ cdt('mesa-dri-drivers') }}      # [linux]
+        - {{ cdt('expat') }}                 # [linux]
```

Comment:
There's a conda-forge package for expat, so why don't we use it instead of the cdt?

Reply:
I'm not sure, I don't properly understand what is kosher for CDT and not, I think the libexpat call was coming from a CDT library, but maybe I just totally misread the traceback.

Reply:
This is a dependency needed by the cdt packages. This should be kept.  It didn't imply expat is installed or needed in end user machines.

Reply:
Ok, @duncanmmacleod please add a comment mentioning that above this line. That'll to make it clear that the CDT `expat` is only needed for `mesa-dri-drivers`.